### PR TITLE
Moved dependencies to dev because they are not needed for build

### PR DESCRIPTION
**Pull request by julien51**
The rule of thumb to add dependencies vs. devDependencies is whether they are required to build or
compile the project.

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [X] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [X] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: smart-contracts/package.json [hunk 1]
```diff
@@ -33,7 +30,10 @@
     "eslint-plugin-standard": "^4.0.0",
     "remix-ide": "^0.7.5",
     "remixd": "^0.1.8-alpha.6",
-    "solhint": "^1.4.0"
+    "shelljs": "^0.8.3",
+    "solhint": "^1.4.0",
+    "zos": "^v2.1.0-rc.0",
```
Comment (nfurfaro) [inline smart-contracts/package.json]: Zos is not required to build, but will be required for automated deploymen, so I'm not sure where that puts it in the current CI setup...

File: smart-contracts/package.json [hunk 2]
```diff
@@ -17,12 +17,9 @@
     "eslint": "^5.7.0",
     "ethereumjs-units": "^0.2.0",
     "openzeppelin-eth": "^2.0.2",
-    "shelljs": "^0.8.3",
     "truffle": "^4.1.14",
     "truffle-contract": "^3.0.6",
-    "web3-utils": "^1.0.0-beta.36",
-    "zos": "^v2.1.0-rc.0",
-    "zos-lib": "^2.0.2"
```
Comment (nfurfaro) [inline smart-contracts/package.json]: Zos-lib is where we import `Initializable.sol`, so I think we need it to compile.

File: smart-contracts/package.json (hunk 1 above)
Comment (julien51) [reply smart-contracts/package.json]: For now, let's keep our out and add it back when we automate deployments..

File: smart-contracts/package.json (hunk 2 above)
Comment (julien51) [reply smart-contracts/package.json]: Something fishy is going on then because build passed :(

Comment (julien51) [reply smart-contracts/package.json]: Something fishy is going on then because build passed :(

**Comment by julien51** [comment]
Ok, I know what that is... since we're not using`--production` on `npm install` or `npm ci`, all dependencies are being included. I'll submit another PR first then ;)

**Pull request closed without merging**